### DISCUSSION TITLE
Using Betterfox with Librewolf

### DISCUSSION THREAD

**Discussion by idontlikecreatingaccounts** [yokoffing/Betterfox#257] [General]
Hi

I'm interested in combining some of the Betterfox features with Librewolf. Because of this, I'm not too worried about the privacy features as I assume those come with Librewolf's default setup. What I am interested in are the Fastfox and Smoothfox. 

It's my understanding that I can just manually type in these settings one by one into about:config and pick the ones I like. That is what I'm doing, but I'm curious if anyone who's more familiar with configuring Firefox than I am knows if this may cause any issues with Librewolf.

Or if you've done something similar yourself and have any advice or experiences, feel free to share.

**Comment by cyrneko** [comment]
Using the Fastfox and Smoothfox configurations should be as simple as drag-n-drop as they don't really interfere with LibreWolf's configuration(s)

**Comment by yokoffing** [comment]
@idontlikecreatingaccounts I created and somewhat maintained [Librewolf overrides](https://github.com/yokoffing/Betterfox/blob/14de7b101d48f15f50df7dd5dbfffefca5b4a855/librewolf.overrides.cfg) for awhile, but I stopped updating it after awhile.

I finally fixed my longtime [issue](https://gitlab.com/librewolf-community/browser/windows/-/issues/251) I had with Librewolf, so I'd be open to exploring it again when I have the time.
